src/main/java/com/softserveinc/ita/mmakoviichuk/models/Template.java

```diff
@@ -0,0 +1,17 @@
+package com.softserveinc.ita.mmakoviichuk.models;
+
+public enum Template {
```

Comment:
it's not the best idea to store the templates in one place, they relate purely to the pages where elements are located